integration_tests/integration/assess-recall.spec.js

```diff
@@ -7,6 +7,7 @@ import {
   getPrisonsResponse,
   getCourtsResponse,
 } from '../mockApis/mockResponses'
+import recallsListPage from '../pages/recallsList'
```

Comment:
Another in passing which I should just try for myself ... the difference between 
`import recallsListPage from '../pages/recallsList'`
and
`const recallsListPage = require('../pages/recallsList')`
has passed me by for now!